app/views/api/v0/articles/show.json.jbuilder

```diff
@@ -7,6 +7,7 @@ json.published_at       @article.published_at
 json.readable_publish_date @article.readable_publish_date
 json.social_image       article_social_image_url(@article)
 json.tag_list           @article.cached_tag_list
+json.tags               @article.cached_tag_list.split(", ")
```

Comment:
Would it be better using `@article.cached_tag_list_array` instead like the `index.json.jbuilder`?
https://github.com/thepracticaldev/dev.to/blob/d66da649dd2978e5bcda868ed5d4cf0df429c54c/app/views/api/v0/articles/index.json.jbuilder#L8